packages/experimental/RadioGroup/src/Radio/RadioExperimental.test.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Radio } from './Radio';
+import * as renderer from 'react-test-renderer';
+import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
+
+describe('Radio component tests', () => {
+  it('Radio default', () => {
+    const tree = renderer.create(<Radio value="key1" label="Default Radio" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Radio disabled', () => {
+    const tree = renderer.create(<Radio disabled value="key1" label="Disabled Radio" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Radio simple rendering does not invalidate styling', () => {
+    checkRenderConsistency(() => <Radio value="key1" label="Default Radio" />, 2);
+  });
+
+  it('Radio re-renders correctly', () => {
+    checkReRender(() => <Radio value="key1" label="Render twice" />);
+  });
+
+  /* Re-Render test for depth 2 is blocked and needs to be revisited later. */
```

Comment:
Can you add more detail on why it's blocked? Otherwise it's hard to know when it should be fixed.

Reply:
Also it'd be better to have this comment before the code that it's referencing so that you have context before reading the code.

Reply:
Makes sense, updated!